### ISSUE TITLE
Gradient Descent Walkthrough

### ISSUE THREAD

**Issue by samogden** [OtterDen-Lab/QuizGenerator#23]
Input: a function f(x1, x2, ....), a learning rate, and a starting point
Output:
| n | location | gradient (d) | update (\alpha * d) |
|---|----------|--------------|---------------------|
| 1 |          |              |                     |
| 2 |          |              |                     |
| 3 |          |              |                     |

Where students need to fill in the different blanks.  Additionally, we should ask "does this seem to converge" based on whether values are monotonically decreasing.

Note: We should have a flag to make this single variable
